src/components/TableList/_table-list.scss

```diff
@@ -1,4 +1,11 @@
-.table-list_error-message {
-  color: red;
-  font-size: 0.5rem;
+.table-list {
+  &_error-message {
+    color: red;
+    font-size: 0.5rem;
+  }
+
+  &_no-models-message {
```

Comment:
Should be `&--no-models-message` at it's a modifier on the list.

Reply:
After reading the documentation I think that this needs to be `&__no-models-message` as it's an element not a modifier.